packages/react-core/src/components/Button/Button.docs.js

```diff
@@ -0,0 +1,16 @@
+import { Button, ButtonVariant, ButtonType } from '@patternfly/react-core';
+import VariantsExample from './examples/ButtonVariants';
+import BlockExample from './examples/BlockButton';
+import LinkExample from './examples/LinkButton';
+
+export default {
+  title: 'Button',
```

Comment:
Would it be okay if I modify this later on to also include an optional markup file for design guidelines?
For example would be displayed here:
https://redhat.invisionapp.com/share/JUNHXEDMVAQ#/screens/314096290

Reply:
I would imagine the website would house that since it is not really React specific.  There is a "description" field that will eventually support markup, and we could potentially add other fields that support markup too (best practices, a11y, etc). I think we should try to keep any docs that apply to both core and react in the website repo.

Reply:
Sounds good, thanks Doug